doc/doxygen/Makefile

```diff
@@ -89,7 +89,7 @@ VENV = ./venv
 SASS = ./node_modules/.bin/sass
```

Comment:
[fix] Can remove this now

Reply:
Addressed 389b813f425409ef2f0cd7c75adebad00d8d5119